views/chooseAll-answer.pug

```diff
@@ -0,0 +1,45 @@
+if answerForm
+	.form-group.col.s12
+		each index in [1,2,3,4]
+			.row
+				.input-field.col.s1
+					input(type='checkbox', name='checkButton', value='mcans'+index, id='mcans'+index)
+					label(for='mcans'+index)
+				.input-field.col.s10
+					input.form-control(type='text', name='mcans'+index, placeholder='Enter Answer Here', required)
+				.input-field.col.s1
+					a.btn-floating.btn-tiny.waves-effect.waves-light.red(onclick='$(this).parent().parent().remove()')
+						i.tiny.material-icons close
+
+	#addMoreAnswersButton
+		a.waves-effect.waves-light.btn(onclick="addMCAnswers('checkbox')") add answer
+
+else if studentQuestionForm
+	form.form-question#chooseAll_answerForm(style='text-align: left;')
+		each answer in question.choices
+			.row
+				input(type='checkbox' name='checkButton', value=answer, id=answer, autofocus)
+				label(for=answer) #{answer}
+
+		button(class='btn btn-lg btn-primary btn-block' type='submit') Submit
+
+else if adminQuestionEdit
+	.form-group.col.s12
+		each option, i in question.choices
+			.row
+				.input-field.col.s1
+					if (question.answer.indexOf(option) > -1)
+						input(type='checkbox', name='checkButton', id=i, value='mcans'+i, checked)
+					else
+						input(type='checkbox', name='checkButton', id=i, value='mcans'+i)
+					label(for=i)
+				.input-field.col.s10
+					input.form-control(type='text', name='mcans'+i, placeholder='Enter New Answer Here', required, style='float:left;' value=option)
+				.input-field.col.s1
+				a.btn-floating.btn-tiny.waves-effect.waves-light.red(onclick='$(this).parent().remove()')
+					i.tiny.material-icons close
+	#addMoreAnswersButton
+		a.waves-effect.waves-light.btn(onclick="addMCAnswers('checkbox')") add answer
+
+block scripts
+	script(src='/js/mc-answer.js')
```

Comment:
choose all question type needs to have its own `.js` file. the 2 types of questions dont need to know of each other's existence, they should be completely separate

Reply:
done